lib/stellar_core_commander/transactor.rb

```diff
@@ -356,6 +356,15 @@ def set_upgrades(protocolversion)
       @process.set_upgrades protocolversion
     end
 
+    Contract None => Any
+    def check_uses_latest_protocol_version
+      latest_protocol_version = @process.info["protocol_version"]
+      current_protocol_version = @process.info["ledger"]["version"]
```

Comment:
I think similarly to `protocol_version`, we could do the field extraction in `process.rb` and just have transactor call that method.